docs/integrations/sources/typeform.md

```diff
@@ -70,6 +70,7 @@ API rate limits \(2 requests per second\): [https://developer.typeform.com/get-s
 
 | Version | Date       | Pull Request                                             | Subject                                                                 |
 |:--------|:-----------|:---------------------------------------------------------|:------------------------------------------------------------------------|
+| 0.1.13  | 2023-06-17 | [00000](https://github.com/airbytehq/airbyte/pull/00000) | Add missing schema fields in `forms`, `themes`, `images` and `workspaces` streams                                                   |
```

Comment:
please update the link to this PR

Reply:
yep, forgot to push commit with this change on Saturday, pushed it now